### ISSUE TITLE
Javadoc fails accessibility requirements for color contrast

### ISSUE THREAD

**Issue by glassfishrobot** [javaee/javaserverfaces-spec#1214]
The Facelets and JavaScript documentation for JSF fail accessibility requirements for color contrast. Sorry I missed this before.

I'll attach the output of the color contrast analyzer for four pages that fail: the main page for the Facelets Tag Library doc, the main page for composite components, the main page for Facelets templating, and one of the JavaScript pages (jsf). You only have to worry about the table rows that are red.

In most cases the colored fonts need to be darker. For the JavaScript pages, it might be sufficient to make the typeface bold for the white-on-blue table headings.
#### Affected Versions
[2.2]

**Comment by glassfishrobot** [comment]
Reported by chaase3

**Comment by glassfishrobot** [comment]
Was assigned to chaase3

**Comment by glassfishrobot** [comment]
chaase3 said:
Whoops, I can't attach files. I will send them in email.

**Comment by glassfishrobot** [comment]
File: [file01-cc.html](https://java.net/jira/secure/attachment/52440/file01-cc.html)
Attached By: @edburns

**Comment by glassfishrobot** [comment]
File: [file02-facelets-ui.html](https://java.net/jira/secure/attachment/52441/file02-facelets-ui.html)
Attached By: @edburns

**Comment by glassfishrobot** [comment]
File: [file03-javascript.html](https://java.net/jira/secure/attachment/52442/file03-javascript.html)
Attached By: @edburns

**Comment by glassfishrobot** [comment]
File: [file04-mainpage.html](https://java.net/jira/secure/attachment/52443/file04-mainpage.html)
Attached By: @edburns

**Comment by glassfishrobot** [comment]
@edburns said:
Updated to avoid A11Y violations.

**Comment by glassfishrobot** [comment]
@edburns said:
Also, I attached a javadoc jar (which you can just unzip with the zip
tool) that has the changes applied.

Kim, I'm assigning the issue to you. Please confirm the changes achieve
compliance and let me know. I'll commit when whe achieve compliance.

Thanks,

Ed

**Comment by glassfishrobot** [comment]
chaase3 said:
Thanks very much, Ed. It's all fixed except for one page:

vdldocs/facelets/f/tld-summary.html

On this page, the equals signs next to the red text are green (it's hard to see between the black and the red) – so the color contrast fails. You might as well make the page black-and-white like the others.

**Comment by glassfishrobot** [comment]
File: [javax.faces-api-2.2-SNAPSHOT-javadoc.jar](https://java.net/jira/secure/attachment/52461/javax.faces-api-2.2-SNAPSHOT-javadoc.jar)
Attached By: @edburns

**Comment by glassfishrobot** [comment]
File: [changebundle.txt](https://java.net/jira/secure/attachment/52462/changebundle.txt)
Attached By: @edburns

**Comment by glassfishrobot** [comment]
@edburns said:
I've just attached another iteration, fixing the problem you mentioned. Please verify.

**Comment by glassfishrobot** [comment]
chaase3 said:
Thanks, Ed! The fix works.

**Comment by glassfishrobot** [comment]
Marked as **fixed** on Wednesday, August 28th 2013, 8:23:21 am

**Comment by glassfishrobot** [comment]
@edburns said:
Kim, please mark this closed when the docs have been published.

**Comment by glassfishrobot** [comment]
@manfredriem said:
Closing resolved issue out

**Comment by glassfishrobot** [comment]
This issue was imported from java.net JIRA JAVASERVERFACES_SPEC_PUBLIC-1214
